src/components/views/rooms/MessageComposerInput.js

```diff
@@ -573,7 +573,7 @@ export default class MessageComposerInput extends React.Component {
         }
 
         // emojioneify any emoji
-        while (true) {
+        do {
             let foundEmoji = false;
```

Comment:
did you test this? I wonder if the delcaration of `foundEmoji` needs to be outside of the loop ...

Reply:
I did not, and you're right, it does.